test/test_util.py

```diff
@@ -35,5 +37,5 @@ def test_cve_scanner(self):
 
         instance_attrs = vars(CVEScanner)["__annotations__"]
         assert (
-            instance_attrs["all_cve_data"] == DefaultDict[ProductInfo, CVEData]
+            instance_attrs["all_cve_data"] == defaultdict[ProductInfo, CVEData]
```

Comment:
Looks like python doesn't like something about this:

```
>       assert (
            instance_attrs["all_cve_data"] == defaultdict[ProductInfo, CVEData]
        ), "Type of all_cve_data has been changed. Make sure it isn't breaking OutputEngine!"
E       TypeError: 'type' object is not subscriptable

test/test_util.py:39: TypeError
```

Reply:
Sorry, that was needlessly vague -- it's failing because the type of all_cve_data no longer matches the test. You might need to either fix where the type of all_cve_data is set outside of the test/ directory, or leave this test as one that can only be fixed when someone else does that.

Reply:
"Type of all_cve_data has been changed. Make sure it isn't breaking OutputEngine!"
This error is actually coming after the moment I changed the cve_scanner.py file to include from __future__.

Reply:
I guess I'll just leave it out of this PR and try it out in a separate one.